server/managers/user.js

```diff
@@ -66,9 +69,19 @@ async function addAnonymousUser(username, passwordHash) {
   await model.updateUserById(user._id, { completedRegistration: true });
 }
 
+async function setUserPermissions(id, requestedPermissions) {
+  const permissions = requestedPermissions || permissionLevel.IS_MANAGER;
+  return new Promise((resolve, reject) => {
```

Comment:
`async` already create a promise so you can just do this:

```js
return model.updateUserById(id, { permissions }, { new: true });
```

Reply:
Cool beans, will fix. 👍